tests/test_git.py

```diff
@@ -204,6 +204,25 @@ def test_git_encoding_error(self):
         self.assertEqual(commit['commit'], 'cb24e4f2f7b2a7f3450bfb15d1cbaa97371e93fb')
         self.assertEqual(commit['message'], 'Calling \udc93Open Type\udc94 (CTRL+SHIFT+T) after startup - performance improvement.')
 
+    def test_git_cr_error(self):
+        """Test if mislocated carriage return chars do not break lines
+
+        In some commit messages, carriage return characters (\r) are found
+        in weird places. They should not be misconsidered as end of line.
+
+        Before fixing, this test raises an exception:
+        "perceval.errors.ParseError: commit expected on line 10"
+
+        """
+
+        commits = Git.parse_git_log_from_file("data/git_bad_cr.txt")
+        try:
```

Comment:
Just to make the error explicit. But I'm removing it anyway, since you seem to prefer that ;-)